client/src/components/teams/SubTeams.vue

```diff
@@ -20,37 +15,28 @@
     <v-col cols="12">
       <v-list v-model:opened="state.open">
         <v-list-group value="Users">
-          <template v-for="team in state.subteams" :key="team.teamName">
-            <v-list-group sub-group no-action :value="team.teamName">
+          <template v-for="subteam in subteamViews" :key="subteam.name">
+            <v-list-group sub-group no-action :value="subteam.subteamId">
               <template v-slot:activator="{ props }">
-                <!--avatar-->
                 <v-list-item v-bind="props">
                   <v-list-item-content>
-                    <!--team name-->
+                    <!--Subteam Info-->
                     <div class="d-flex align-center">
-                      {{ team.teamName }}
+                      {{ subteam.name }}
                       <span class="km-text rounded-lg ml-auto text-no-wrap pa-1 bg-success">
-                        {{ team.totalKM }}
+                        {{ subteam.totalKm }}
```

Comment:
Since the totalKM isnt implemented properly yet, we can comment this out for now.